packages/web-app-files/src/components/SideBar/Audio/AudioMetaPanel.vue

```diff
@@ -99,17 +99,27 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+#files-sidebar-panel-audio {
+  background-color: var(--oc-role-surface-container);
+  border-radius: 5px;
```

Comment:
You could use `oc-rounded` as a class on the panel instead of hardcoding the pixels here.